лаба.py

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
remove

Reply:
done

Reply:
I do not think so. remove the file

Reply:
I am trying to delete it and commit the changes but it's still here..............

Reply:
fixed!